apps/src/craft/designer/blocks.js

```diff
@@ -0,0 +1,720 @@
+  bricks: i18n.miniBlockBricks(),
+  logAcacia: i18n.miniBlockLogAcacia(),
+  logBirch: i18n.miniBlockLogBirch(),
+  logJungle: i18n.miniBlockLogJungle(),
+  logOak: i18n.miniBlockLogOak(),
+  logSpruce: i18n.miniBlockLogSpruce(),
+  planksAcacia: i18n.miniBlockPlanksAcacia(),
+  planksBirch: i18n.miniBlockPlanksBirch(),
+  planksJungle: i18n.miniBlockPlanksJungle(),
+  planksOak: i18n.miniBlockPlanksOak(),
+  planksSpruce: i18n.miniBlockPlanksSpruce(),
+  cobblestone: i18n.miniBlockCobblestone(),
+  sandstone: i18n.miniBlockSandstone(),
+  wool: i18n.miniBlockWool(),
+  redstoneDust: i18n.miniBlockRedstoneDust(),
+  lapisLazuli: i18n.miniBlockLapisLazuli(),
+  ingotIron: i18n.miniBlockIngotIron(),
+  ingotGold: i18n.miniBlockIngotGold(),
+  emerald: i18n.miniBlockEmerald(),
+  diamond: i18n.miniBlockDiamond(),
+  coal: i18n.miniBlockCoal(),
+  bucketWater: i18n.miniBlockBucketWater(),
+  bucketLava: i18n.miniBlockBucketLava(),
+  gunPowder: i18n.miniBlockGunPowder(),
+  wheat: i18n.miniBlockWheat(),
+  potato: i18n.miniBlockPotato(),
+  carrots: i18n.miniBlockCarrots(),
+  milk: i18n.miniBlockMilk(),
+  egg: i18n.miniBlockEgg(),
+  poppy: i18n.miniBlockPoppy(),
+  sheep: i18n.miniBlockSheep(),
+};
+
+const miniBlocks = Object.keys(miniBlocksToDisplayText).sort();
+
+const soundsToDisplayText = {
+  dig_wood1: i18n.soundTypeDig_wood1(),
+  stepGrass: i18n.soundTypeStepGrass(),
+  stepWood: i18n.soundTypeStepWood(),
+  stepStone: i18n.soundTypeStepStone(),
+  stepGravel: i18n.soundTypeStepGravel(),
+  stepFarmland: i18n.soundTypeStepFarmland(),
+  failure: i18n.soundTypeFailure(),
+  success: i18n.soundTypeSuccess(),
+  fall: i18n.soundTypeFall(),
+  fuse: i18n.soundTypeFuse(),
+  explode: i18n.soundTypeExplode(),
+  placeBlock: i18n.soundTypePlaceBlock(),
+  collectedBlock: i18n.soundTypeCollectedBlock(),
+  bump: i18n.soundTypeBump(),
+  punch: i18n.soundTypePunch(),
+  fizz: i18n.soundTypeFizz(),
+  doorOpen: i18n.soundTypeDoorOpen(),
+  minecart: i18n.soundTypeMinecart(),
+  sheepBaa: i18n.soundTypeSheepBaa(),
+  chickenHurt: i18n.soundTypeChickenHurt(),
+  chickenBawk: i18n.soundTypeChickenBawk(),
+  cowHuff: i18n.soundTypeCowHuff(),
+  cowHurt: i18n.soundTypeCowHurt(),
+  cowMoo: i18n.soundTypeCowMoo(),
+  cowMooLong: i18n.soundTypeCowMooLong(),
+  creeperHiss: i18n.soundTypeCreeperHiss(),
+  ironGolemHit: i18n.soundTypeIronGolemHit(),
+  metalWhack: i18n.soundTypeMetalWhack(),
+  zombieBrains: i18n.soundTypeZombieBrains(),
+  zombieGroan: i18n.soundTypeZombieGroan(),
+  zombieHurt: i18n.soundTypeZombieHurt(),
+};
+
+const allSounds = Object.keys(soundsToDisplayText);
+
+const entityTypesToDisplayText = {
+  Player: i18n.entityTypePlayer(),
+  sheep: i18n.entityTypeSheep(),
+  zombie: i18n.entityTypeZombie(),
+  ironGolem: i18n.entityTypeIronGolem(),
+  creeper: i18n.entityTypeCreeper(),
+  cow: i18n.entityTypeCow(),
+  chicken: i18n.entityTypeChicken(),
+};
+
+const ENTITY_TYPES = Object.keys(entityTypesToDisplayText);
```

Comment:
Any rationale behind why sometimes we go all caps (ENTITY_TYPES) and sometimes we don't (allSounds)